Userland/Libraries/LibGUI/ComboBox.cpp

```diff
@@ -25,6 +25,7 @@ class ComboBoxEditor final : public TextEditor {
 
```

Comment:
Regarding your commit message, as this patch doesn't touch `KeyboardSettings` you also wouldn't use that prefix. Instead use `LibGUI: `

Reply:
I got this when i try to reword or amend the commit...
```
Running Meta/lint-ci.sh to ensure changes will pass linting on CI......(no files to check)Skipped
Running Meta/lint-ports.py to ensure changes will pass linting on CI...(no files to check)Skipped
Lint commit message to ensure it will pass the commit linting on CI.......................Failed
- hook id: meta-lint-commit
- exit code: 1
```
is this some project specific thing?Sorry for this noob questions😂